specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json

```diff
@@ -9892,6 +11960,23 @@
       "type": "string",
       "in": "path",
       "x-ms-skip-url-encoding": true
+    },
+    "IncludeResponse": {
```

Comment:
If this belongs to the Async query logs and you removed it, please remove this as well.